ppr-ui/src/components/mhrRegistration/HomeOwners/FractionalOwnership.vue

```diff
@@ -12,14 +12,17 @@
       <v-text-field
         :id="`interest-type-group-${groupId}`"
         label="Interest Type (Optional)"
+        :filled="!editingGroup"
+        :class="{ backgroundWhite: editingGroup }"
         v-model="fractionalData.interest"
         :data-test-id="`interest-type-field-group-${groupId}`"
       />
       <div class="owner-fractions">
         <v-text-field
           :id="`fraction-amount-group-${groupId}`"
           label="Amount Owned by this Group"
-          filled
+          :filled="!editingGroup"
```

Comment:
These two changes cause the following:

<img width="980" alt="Screen Shot 2022-09-19 at 10 44 27 AM" src="https://user-images.githubusercontent.com/2333290/191081522-0a8b6a29-28c0-4ceb-85ab-ff79f4c8dbf9.png">

Reply:
There is a styling issue.. The requirement is correct.. I may need assistance fixing this one...